src/rpc/handlers/Subscribe.cpp

```diff
@@ -442,8 +442,7 @@ doUnsubscribe(Context const& context)
     if (request.contains("books"))
         unsubscribeToBooks(books, context.session, *context.subscriptions);
 
-    boost::json::object response = {{"status", "success"}};
-    return response;
+    return boost::json::object{};
```

Comment:
This is for issue #354. We did not remove this `status` back then but it seems like that's what is needed to match `clio`'s `subscribe` behaviour and `rippled` as well.